app/controllers/posts_controller.rb

```diff
@@ -1,27 +1,48 @@
 class PostsController < ApplicationController
+  
+  def index
+    @posts = Post.all.order(created_at: :desc)
```

Comment:
Looks really concise, could you explain how it works and if it could still operate without the all method?